task_either.ts

```diff
@@ -141,6 +141,13 @@ export function widen<J>(): <A, B>(
   return identity;
 }
 
+export function fold<L, R, B>(
```

Comment:
This looks good but I'd like to get into the habit of documenting new functions as they are added. Could you add some text explaining the function along with an example? You can look at [affect.ts for ](https://github.com/nullpub/fun/blob/documentation/affect.ts) for examples of the current documentation format.